### ISSUE TITLE
Update columns related to FARE_ADJ

### ISSUE THREAD

**Issue by jawadmhoque** [jawadmhoque/transit_ridership_decline#10]
Bus_FARE_TOTAL | Bus_FARE_TOTAL_diff | Bus_FARE_TOTAL_log | PCT_Bus_FARE_TOTAL_rel_diff | Bus_FARE_TOTAL_scale
and the Relevant Rail ones. 

Note: What was previously called FARE_TOTAL is currently FARE_ADJ in the new estimation file (Factors and RIdership Data>Estimation_File>estimation_file.csv)

**Comment by gregerhardt** [comment]
In the estimation, I am using FARE_per_UPT_2018.  Is that the right field?

**Comment by gregerhardt** [comment]
Can this issue be closed?

**Comment by gregerhardt** [comment]
@jawadmhoque  Can you have one more look at the fare data?  Looking at the FAC tables, the numbers look too high.  For example, NYC bus has an average fare of about $8.00.  These values should be more like $1 to $2 in most places, sometimes less.  Rail might be higher.  

![image](https://user-images.githubusercontent.com/5774986/70866610-ce1faa00-1f39-11ea-9ebc-0dc0b3b418c9.png)

**Comment by vsgo222** [comment]
> 
> 
> Can this issue be closed?

Yes. It has been incorporated in the use of Est6 --> UPT_FAC_files

**Comment by jawadmhoque** [comment]
> In the estimation, I am using FARE_per_UPT_2018. Is that the right field?

It is the right field. 
I am looking at the Fare data cleanup again. I think such abnormally large values come from adding up the modes that are not present in the UPT data. I checked the NYC data fare from the original NTD excel files and our estimation file, and they are matching.

**Comment by gregerhardt** [comment]
I see that NYC matches the original, but there is still a weird spike in NYC in 2006.  We should link a set of overrides to the main estimation file to smooth this out, as well as any others that are weird.
